worker-document/src/main/java/com/hpe/caf/worker/document/scripting/specs/InlineScriptSpec.java

```diff
@@ -25,10 +26,12 @@
 public final class InlineScriptSpec extends AbstractScriptSpec
 {
     private final String script;
+    private final ScriptEngineType engineType;
```

Comment:
You're adding this new variable to this and all the other concrete implementations of `AbstractScriptSpec`, but you haven't updated any of the `equals()` or `hashCode()` methods of these classes.  Also, why add this to each of the concrete implementations instead of further up the inheritance hierarchy to share the code?

Reply:
This comment hasn't been addressed:

> why add this to each of the concrete implementations instead of further up the inheritance hierarchy to share the code?

Reply:
I've made the change in commit 096e6f2.